schema/DoctrineMigrations/Version20231218161807.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Ivoz\Core\Infrastructure\Persistence\Doctrine\LoggableMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20231218161807 extends LoggableMigration
+{
+    public function getDescription(): string
+    {
+        return 'Added accessCredentials in Notification Templates';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE NotificationTemplates CHANGE type type VARCHAR(25) NOT NULL COMMENT \'[enum:voicemail|fax|limit|lowbalance|invoice|callCsv|maxDailyUsage|accessCredentials]\'');
+        $this->addSql("INSERT INTO NotificationTemplates (name, type) VALUES ('Generic Access Credentials', 'accessCredentials')");
```

Comment:
Prepend "notification template" to name column (in order to be coherent with remaining entries)